### PR TITLE
sockperf: add configure flag to enable extra api

### PR DESCRIPTION
**Pull request by rafiw**
Signed-off-by: Rafi Wiener <rafiw@mellanox.com>

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -72,6 +72,21 @@ SP_CHECK_CXXFLAGS_APPEND([OUR_CXXFLAGS], [\
...
+                   AC_HELP_STRING([--enable-extra-api],
+                                  [enable vma extra api support, default=no]))
+
+AS_IF([test "x$have_extra_api" = "xyes"],
+      [AC_CHECK_HEADER([mellanox/vma_extra.h],
+                       [AC_MSG_RESULT(extra api enabled)]
+                       [AC_DEFINE([USING_VMA_EXTRA_API],[1],
+                                  [using vma extra api])],
+                       [AC_MSG_ERROR([vma_extra.h file not found])])],
+      [AC_MSG_RESULT(extra api disabled)])
```
Comment (igor-ivanov) [inline configure.ac]: @rafiw please consider removing https://github.com/Mellanox/sockperf/blob/sockperf_v2/configure.ac#L143 and using following construction as
AC_MSG_CHECKING(
    [if VMA Extra API is enabled])
...
    AC_MSG_RESULT([yes])
else
    AC_MSG_RESULT([no])

**Comment by rafiw** [comment]
added by #76 
https://github.com/Mellanox/sockperf/blob/sockperf_v2/configure.ac#L143

**Comment by rafiw** [comment]
@igor-ivanov updated

**Comment by rafiw** [comment]
@igor-ivanov do you think the default should yes?

**Comment by igor-ivanov** [comment]
@rafiw I think that default should be `yes` but configure should not fail by error if Mellanox VMA is not found. Let display just information (yes/no). In fact this behavior should be the same as in https://github.com/Mellanox/sockperf/blob/sockperf_v2/configure.ac#L143

**Comment by igor-ivanov** [comment]
merged (see https://github.com/Mellanox/sockperf/commit/cd5f87c)

**Pull request closed without merging**